src/ngraph_builder.cc

```diff
@@ -1491,6 +1491,25 @@ static Status TranslateFusedBatchNormGradOp(
   return Status::OK();
 }
 
+static Status TranslateSigmoidGradOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  shared_ptr<ng::Node> ng_delta;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input, &ng_delta));
+
+  auto ng_mul = std::make_shared<ng::op::Multiply>(ng_input, ng_delta);
+  ng::Shape input_shape = ng_input->get_shape();
```

Comment:
ng nodes have support for basic arithmetic ops, so one can use: `ng_input * ng_delta`

Reply:
Oh, good to know. I will make all the changes accordingly. Thanks!